tests/cxx/badinc.cc

```diff
@@ -1894,42 +1894,6 @@ int main() {
   local_d1_copy_class = D1CopyClassFn(I12);
   local_d1_copy_class.a();
 
-  // Test (templated) function pointers and method pointers.
-  // IWYU: I1_Class needs a declaration
-  // IWYU: I1_Enum is...*badinc-i1.h
-  // IWYU: I1_Function is...*badinc-i1.h
-  I1_Enum (*local_fn_ptr)(I1_Class*) = &I1_Function;
-  // IWYU: I1_Class is...*badinc-i1.h
-  int (*static_method_ptr)() = &I1_Class::s;
-  // IWYU: I1_Struct is...*badinc-i1.h
-  // IWYU: I1_Struct needs a declaration
-  // IWYU: I1_TemplateMethodOnlyClass is...*badinc-i1.h
-  int (*tpl_fn_ptr)() = &I1_TemplateMethodOnlyClass<I1_Struct>::stat;
```

Comment:
`I1_TemplateMethodOnlyClass::stat` seems to be not used anymore. Do you think it's worth removing?